scripts/push_psnr_results.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
```

Comment:
Adding the licence header and a brief description of what this does would be nice

Reply:
good idea, will do